docs/Competitions.md

```diff
@@ -42,13 +42,13 @@ Install all the dependencies:
 
     cd codalab-worksheets
     virtualenv -p python3.6 venv
```

Comment:
Don't need to git clone or make a venv anymore.  The whole point is that you can now just `pip install codalab` and type `cl-competitiond ...`

Reply:
But should check that this works.

Reply:
The only thing is that users may not have all the package requirements installed in their local machine.

Reply:
That's true. I installed it in local mode when I was testing the code change. It didn't install the dependencies. I'll update the code.